### PR TITLE
Mapper to presenter

### PR DESCRIPTION
**Pull request by s12chung**
updating the code is like:

``` ruby
class Mapper
 include CsvRowModel::Import::Mapper
 maps_to ImportRowModel

 # implementation
end

mapper = CsvRowModel::Import::File.new(file_path, Mapper).next
```

to:

``` ruby
class ImportRowModel
  # normal ImportRowModel code

  presenter do
    # Mapper implementation is copy and pasted here
  end
end

row_model = CsvRowModel::Import::File.new(file_path, ImportRowModel).next
presenter = row_model.presenter
```

or you can read the README: https://github.com/FinalCAD/csv_row_model/blob/e375dc28afc0fb1c9b0ec552a27be2987118394c/README.md#presenter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,16 +66,70 @@ row_model.name # => "SOME PROJECT NAME"
...
+## Presenter
+For complex rows, you can wrap your `RowModel` with a presenter:
+
+```ruby
+class ProjectImportRowModel < ProjectRowModel
+  include CsvRowModel::Import
+
+  # same as above
+
+  presenter do
```
Comment (ale7714) [inline README.md:83]: 👍🏻 I think is easier to read the import model... Maybe some clases can become really big though

**Comment by ale7714** [comment]
✅ read

File: spec/csv_row_model/concerns/deep_class_var_spec.rb [hunk 2]
```diff
@@ -26,6 +28,95 @@ class ClassWithFamily < Parent; end
...
+          Class.new do
+            include CsvRowModel::Model
+            def self.deep_class_module; CsvRowModel::Model end
+
+            csv_string_model { validates :string1, presence: true }
+          end
+        end
+
+        it "adds csv_string_model_class validations" do
+          expect(klass.csv_string_model_class.new(string1: "blah").valid?).to eql true
```
Comment (joel) [inline spec/csv_row_model/concerns/deep_class_var_spec.rb]: `expect(klass.csv_string_model_class.new(string1: "blah")).to be_valid`

File: spec/csv_row_model/concerns/deep_class_var_spec.rb [hunk 3]
```diff
@@ -26,6 +28,95 @@ class ClassWithFamily < Parent; end
...
+          expect(klass.csv_string_model_class.new(string1: "").valid?).to eql false
```
Comment (joel) [inline spec/csv_row_model/concerns/deep_class_var_spec.rb]: `expect(klass.csv_string_model_class.new(string1: "")).to_not be_valid`

Comment (s12chung) [inline spec/csv_row_model/concerns/deep_class_var_spec.rb]: same thing to me :D. thanks for the change :)

**Pull request merged**